specification/azsadmin/resource-manager/deployment/Microsoft.Deployment.Admin/preview/2018-07-01/FileContainer.json

```diff
@@ -0,0 +1,363 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-07-01",
+    "title": "DeploymentAdminClient",
+    "description": "Deployment Admin Client."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Deployment.Admin/locations/global/fileContainers": {
+      "get": {
+        "x-ms-examples": {
+          "Return file containers list.": {
+            "$ref": "./examples/FileContainer/List.json"
+          }
+        },
+        "description": "Returns an array of file containers.",
+        "tags": [
+          "FileContainers"
+        ],
+        "operationId": "FileContainers_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/FileContainersList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Deployment.Admin/locations/global/fileContainers/{fileContainerId}": {
+      "get": {
+        "x-ms-examples": {
+          "Return the file container details.": {
+            "$ref": "./examples/FileContainer/Get.json"
+          }
+        },
+        "description": "Retrieves the specific file container details.",
+        "tags": [
+          "FileContainers"
+        ],
+        "operationId": "FileContainers_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/FileContainerIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/FileContainer"
+            }
+          },
+          "404": {
```

Comment:
All the 404 should be repalced with Default error response

Reply:
like this https://github.com/Azure/azure-rest-api-specs/blob/master/specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-12-01/apimapis.json#L100

Reply:
added Default error responses . Please review it .